llama_index/vector_stores/qdrant.py

```diff
@@ -74,7 +87,9 @@ def __init__(
         aclient: Optional[Any] = None,
         url: Optional[str] = None,
         api_key: Optional[str] = None,
-        batch_size: int = 100,
+        batch_size: int = 64,
```

Comment:
can you keep the default batch_size 100

Reply:
Qdrant client's default is 64. Also, we had quite some users hitting the 32 MB/request limit when this was 100.

Reply:
ic, thanks!